gef.py

```diff
@@ -6760,55 +6749,136 @@ def do_invoke(self, argv):
             info("No Tcache in this version of libc")
             return
 
-        arena = GlibcArena("*{:s}".format(argv[0])) if len(argv) == 1 else get_main_arena()
-
-        if arena is None:
-            err("Invalid Glibc arena")
+        current_thread = gdb.selected_thread()
+        if current_thread is None:
+            err("Couldn't find current thread")
             return
 
-        # Get tcache_perthread_struct for this arena
-        heap_base = HeapBaseFunction.heap_base()
-        if heap_base is None:
-            err("No heap section")
-            return
-        addr = heap_base + 0x10
-
-        gef_print(titlify("Tcachebins for arena {:#x}".format(int(arena))))
-        for i in range(GlibcArena.TCACHE_MAX_BINS):
-            if get_libc_version() < (2, 30):
-                count = ord(read_memory(addr + i, 1))
+        threads = gdb.selected_inferior().threads()
```

Comment:
might as well sort here, that way we get `tids` sorted as well when using `all`

Reply:
Addressed the rest of the comments in ecf9a5c8ca0f8b08e1ae63ce209778e3e8c551fd